Hackney.Shared.Finance/Boundary/Response/PagingParametersDomain.cs

```diff
@@ -0,0 +1,9 @@
+﻿
+namespace Hackney.Shared.Finance.Boundary.Response
+{
+    public abstract class PagingParametersDomain
```

Comment:
Same as mentioned previously. I believe these classes shouldn't be abstract classes.